service/operator/service.go

```diff
@@ -123,6 +126,11 @@ func (s *Service) addFunc(obj interface{}) {
 		s.Logger.Log("error", "could not convert to azuretpr.CustomObject")
 	}
 
+	_, err := azure.NewClients(s.AzureConfig)
+	if err != nil {
+		s.Logger.Log("error", "could not create azure api clients '%#v'")
+	}
```

Comment:
I agree that is business logic but I don't think we need auto renewing tokens yet. I've created #9 to track this.